### DISCUSSION TITLE
Improvement to Minimap: Colour segment functions, better visualisation and location indicator

### DISCUSSION THREAD

**Discussion by elias-Mimetrik** [microsoft/vscode-discussions#1985] [Extension Show and Tell]
The minimap is essential in helping me navigate, especially to know where I am at the code. Why not use colours to segment functions (only functions) like so: It makes the code much easier to see and digest:

![Screenshot from 2024-10-14 16:18:41](https://github.com/user-attachments/assets/6ce6bf10-d204-4302-990b-7ec132ae23d9)

**Comment by gjsjohnmurray** [comment]
Are you developing an extension? If not, this is the wrong place. Please see pinned thread https://github.com/microsoft/vscode-discussions/discussions/5

**Reply by elias-Mimetrik** [reply]
Thank you. I'm not developing an extension, it's only a suggestion for extension/theme developers. Where is the best place to post it please?
